utils.py

```
@@ -0,0 +1,21 @@
+
+def slow_hash_password(password):
+    time.sleep(2)
+    return password
+
+def send_admin_alert(msg):
+    print(f"[ALERT] {msg}")
+
+def get_usernames(users):
+    result = []
+    for user in users:
+        for i in range(1):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)